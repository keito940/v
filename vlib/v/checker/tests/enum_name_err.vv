enum color {
    green
    yellow
}
fn main(){
	println('hello')
}
