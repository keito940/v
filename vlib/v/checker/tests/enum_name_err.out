vlib/v/checker/tests/enum_name_err.v:1:6: error: enum name `color` must begin with a capital letter
    1 | enum color {
      |      ~~~~~
    2 |     green
    3 |     yellow
